packages/react-core/src/demos/Animations/examples/Animations.tsx

```diff
@@ -0,0 +1,1586 @@
+                  variant="danger"
+                  headingLevel="h5"
+                  title="Critical alert regarding control plane"
+                />
+                <NotificationDrawerListItemBody>
+                  This is a long description to show how the title will wrap if it is long and wraps to multiple lines.
+                </NotificationDrawerListItemBody>
+              </NotificationDrawerListItem>
+              <NotificationDrawerListItem variant="warning">
+                <NotificationDrawerListItemHeader variant="warning" headingLevel="h5" title="Warning alert" />
+                <NotificationDrawerListItemBody>
+                  This is a warning notification description.
+                </NotificationDrawerListItemBody>
+              </NotificationDrawerListItem>
+            </NotificationDrawerList>
+          </NotificationDrawerGroup>
+        </NotificationDrawerBody>
+      </NotificationDrawer>
+    );
+
+    return (
+      <Card>
+        {header}
+        {body}
+        <Divider />
+        {drawer}
+      </Card>
+    );
+  };
+
+  const DetailsCard: FunctionComponent = () => (
+    <Card>
+      <CardTitle>
+        <Title headingLevel="h4" size="xl">
+          Details
+        </Title>
+      </CardTitle>
+      <CardBody>
+        <DescriptionList>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Cluster API Address</DescriptionListTerm>
+            <DescriptionListDescription>
+              <a href="#">https://api1.devcluster.openshift.com</a>
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Cluster ID</DescriptionListTerm>
+            <DescriptionListDescription>63b97ac1-b850-41d9-8820-239becde9e86</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Provide</DescriptionListTerm>
+            <DescriptionListDescription>AWS</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>OpenShift Version</DescriptionListTerm>
+            <DescriptionListDescription>4.5.0.ci-2020-06-16-015028</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Update Channel</DescriptionListTerm>
+            <DescriptionListDescription>stable-4.5</DescriptionListDescription>
+          </DescriptionListGroup>
+        </DescriptionList>
+      </CardBody>
+      <Divider />
+      <CardFooter>
+        <a href="#">View Settings</a>
+      </CardFooter>
+    </Card>
+  );
+
+  const detailStatusEvents = (
+    <Grid hasGutter>
+      <GridItem span={4}>
+        <DetailsCard />
+      </GridItem>
+      <GridItem span={4}>
+        <CardStatus />
+      </GridItem>
+      <GridItem span={4}>
+        <EventsCard />
+      </GridItem>
+    </Grid>
+  );
```

Comment:
So that the cards don't get so narrow, can you use props to adjust the columns at narrow widths instead? The 3 across cards could stack when the screen is too narrow. 
![image](https://github.com/user-attachments/assets/88d174d2-3a3f-4c76-9ba0-61270efa0004)